nmtwizard/framework.py

```diff
@@ -978,8 +1017,23 @@ def preprocess_into_model(
         config["build"] = build_info
 
         # Build and push the model package.
-        objects = {"data": data_dir}
-        bundle_dependencies(objects, config, local_config)
+        if parent_model_type == "checkpoint" and model_type == "standalone":
+            objects = {"standalone_data": data_dir}
+            config["data"] = {
+                "sample": config.get("build", {}).get("sentenceCount")
```

Comment:
I think you could set this to 0 so that all the included data are used by default?

Reply:
I thought about it, but 0 only works with "*" distribution.
So I thought it could be bad/misleading for the user, in case he forgets to change it for his own distribution.
But I can change if you prefer.

Reply:
Ok, then it's probably fine like this.